_layouts/ontology_detail.html

```diff
@@ -46,13 +46,15 @@ <h1>
   {% endif %}
 
   {% if page.preferredPrefix %}
+  {% if page.preferredPrefix|upper != page.preferredPrefix %}
```

Comment:
you could combine this with the previous if statement using `and` for brevity

Reply:
Cheers